src/main/java/org/folio/circulation/storage/ItemByInstanceIdFinder.java

```diff
@@ -54,7 +55,7 @@ public ItemByInstanceIdFinder(GetManyRecordsClient holdingsStorageClient,
 
       Set<String> holdingsIds = holdingsRecords.toKeys(byId());
 
-      return itemRepository.findByQuery(exactMatchAny("holdingsRecordId", holdingsIds));
+      return itemRepository.findByQuery(exactMatchAny("holdingsRecordId", holdingsIds), tenThousand());
```

Comment:
Given this limit is only used here, I suggest defining it here rather than in the PageLimit class.

Reply:
Thanks for suggestions, implement solution without pageLimit.